mvp_example/src/main/java/main/presenter/Presenter.kt

```diff
@@ -0,0 +1,112 @@
+package main.presenter
+
+import android.util.Log
+import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.LifecycleObserver
+import androidx.lifecycle.OnLifecycleEvent
+import com.distillery.android.domain.ToDoRepository
+import com.distillery.android.domain.models.ToDoModel
+import kotlinx.coroutines.CoroutineExceptionHandler
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.withContext
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.InternalCoroutinesApi
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.collect
+import main.view.TodoListAdapter
+import org.koin.core.KoinComponent
+import org.koin.core.inject
+import kotlin.coroutines.CoroutineContext
+import kotlin.properties.Delegates
+
+@Suppress("VariableNaming")
+class Presenter(
```

Comment:
Not sure suppress for the whole class is a good idea, if that's because of TAG - just move it to const to the file itself
```suggestion
private const val TAG = "TodoPresenter"

class Presenter(
```

Reply:
Removed unnecesary supressions